packages/neos-ui/src/Containers/PrimaryToolbar/style.css

```diff
@@ -7,6 +7,8 @@
     &:after {
         clear: both;
     }
+
+    z-index: 9999;
```

Comment:
Would be nice to extract all z-index constants into variables, to make the usage of them more predictable, like we did in the old UI.